### ISSUE TITLE
[FromQuery] enum parameter with default value is shown as string with no default value.

### ISSUE THREAD

**Issue by thomaslevesque** [domaindrivendev/Swashbuckle.AspNetCore#767]
I have a controller method like this:

```C#
[HttpGet("")]
[SwaggerResponse(200, typeof(CollectionResponse<Foo>), "Returns the list of foos.")]
public async Task<IActionResult> GetFoosAsync([FromQuery] FooListFilter filter = FooListFilter.All)
```

Where `FooListFilter` is an enum.

In swagger.json, the `filter` parameter is described like this:

```json
{
    "name": "filter",
    "in": "query",
    "description": "The filter",
    "required": false,
    "type": "string",
    "enum": [
        "all",
        "mine",
        "others"
    ]
}
```

As you can see, the default value is not included.

(also, the possible values are not shown in the UI, but I think it's a problem with swagger-ui rather than Swashbuckle, since the values are included in swagger.json)

**Comment by domaindrivendev** [comment]
Fixed by 85829dcc3084742db839cb9c7d6df0a40fc62dfa. Will be generally available with the 3.0.0 release (coming soon). In the meantime, feel free to pull down the latest preview on myget.org and try it out.

**Comment by thomaslevesque** [comment]
Thanks!
